locations/spiders/Amazons.py

```diff
@@ -0,0 +1,130 @@
+import scrapy
+from scrapy.linkextractors import LinkExtractor
+from scrapy.spiders import CrawlSpider, Rule
+import  requests
+from locations.items import  GeojsonPointItem
+
+User_Agent = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.157 Safari/537.36"
+
+
+days = ["Monday","Tuesday","Wednesday","Thursday","Friday","Saturday","Sunday"]
+days_short =["Mon","Tue","Wed","Thur","Fri","Sat","Sun"]
+days_want = ["Mo","Tu","We","Th","Fr","Sa","Su"]
+days_dict = {i:n for n,i in enumerate(days_want)}
+days_rev_dict = {n:i for n,i in enumerate(days_want)}
+class AmazonsSpider(CrawlSpider):
+    name = 'Amazons'
```

Comment:
Please use a lower case name.

```suggestion
    name = 'amazon_retail'
```